cli/file_fetcher.rs

```diff
@@ -533,6 +538,26 @@ fn filter_shebang(bytes: Vec<u8>) -> Vec<u8> {
   }
 }
 
+fn check_cache_blacklist(url: &Url, black_list: &[String]) -> bool {
```

Comment:
Should the `black_list` argument be a `Vec<Url>` instead?

Reply:
we never utilize any of the `Url` properties by elements in the list, so I thought that maybe strings would be better (as in  --allow-read whitelist)

but I agree that `Vec<Url>` would be more clean, and parsing few urls wouldn't harm performance

Reply:
Yea, I guess it's not a big deal.

I'm ready to land this patch.... but maybe you could add a short description of this feature to the website/manual.md somewhere?

Reply:
Sure